README.md

```diff
@@ -195,7 +184,8 @@ Run "very_good help <command>" for more information about a command.
 [testing_link]: https://flutter.dev/docs/testing
 [very_good_analysis_badge]: https://img.shields.io/badge/style-very_good_analysis-B22C89.svg
 [very_good_analysis_link]: https://pub.dev/packages/very_good_analysis
-[very_good_core_link]: doc/very_good_core.md
+[very_good_core_link]: site/docs/templates/core.md
+[migrating_link]: site/docs/resources/migrating_to_0_10_0.md
```

Comment:
```suggestion
[new_syntax_link]: site/docs/resources/migrating_to_0_10_0.md
```

Reply:
see [com](https://github.com/VeryGoodOpenSource/very_good_cli/pull/621/files#r1069931625) for reasoning re: migration vs. syntax change